### ISSUE TITLE
error in bibtexStr parsing when author name contains "and"

### ISSUE THREAD

**Issue by azazellochg** [I2PC/scipion#355]
I have the following bibtexStr:
```bash
@Article{Borland1990,
   author = {Borland, L. and van Heel, M.},
   title = {Classification of image data in conjugate representation spaces},
   journal = {J.Opt.Soc.Am. A},
   year = {1990},
   volume = {7},
   issue = {4}
   pages = {601--610},
   doi = {http://dx.doi.org/10.1364/JOSAA.7.000601}
 }
```
The problem is that in GUI it shows: _Borl, et.al, J.Opt.Soc.Am. A, 1990_. I guess that "and" in the author name is recognized as a "keyword" that separates different authors in the list.

**Comment by delarosatrevin** [comment]
I think this is how the bibtex library that we are using works.

**Comment by azazellochg** [comment]
@delarosatrevin , I have checked manually with bibtexparser and the dictionary items are fine. The problem is that [here](https://github.com/I2PC/scipion/blob/v1.0/pyworkflow/protocol/protocol.py#L1377)  **split('and')** is used.

**Comment by azazellochg** [comment]
closed in de0fce8
